azext_iot/_help.py

```diff
@@ -1197,7 +1197,8 @@
             IoT Device Provisioning Service '{dps_name}' in the resource group
             '{resource_group_name}' with provisioning status 'disabled',
             device id '{device_id}', initial twin properties '{"location":{"region":"US"}}',
-            initial twin tags '{"version":"1"}', and device information '{"color":"red"}'.
+            initial twin tags '{"version":"1"}', and device information '{"color":"red"}'
+            Please provide certificate format using Base64 ASCII encoding and the certificate should start with '-----BEGIN CERTIFICATE-----' and end with '-----END CERTIFICATE-----''.
```

Comment:
Have you tested without the BEGIN CERTIFICATE and END CERTIFICATE  sections ? Just a base64 payload to see if it works?

Reply:
yep this one actually work for the service call